mods/seasonal/statuses.js

```diff
@@ -654,6 +654,31 @@ let BattleStatuses = {
 			this.add(`c|@moo|/me moo`);
 		},
 	},
+	morfent: {
+		noCopy: true,
+		onStart: function () {
+			this.add(`c|@Morfent ( _̀> ̀)|──────▀█████▄──────▲`);
+			this.add(`c|@Morfent ( _̀> ̀)|───▄███████████▄──◀█▶`);
+			this.add(`c|@Morfent ( _̀> ̀)|─────▄████▀█▄──────█`);
+			this.add(`c|@Morfent ( _̀> ̀)|───▄█████████████████▄ -I`);
+			this.add(`c|@Morfent ( _̀> ̀)|─▄█████.▼.▼.▼.▼.▼.▼.▼  -cast`);
+			this.add(`c|@Morfent ( _̀> ̀)|▄███████▄.▲.▲.▲.▲.▲.▲  -magic`);
+			this.add(`c|@Morfent ( _̀> ̀)|█████████████████████▀▀-shitpost`);
+		},
+		onFaint: function () {
+			// Morfent returns something so im doing it this way
+			let endquote = this.random(3);
+			if (endquote === 1) {
+				this.add(`c|@Morfent ( _̀> ̀)|Hacking claims the lives of over 2,000 registered laddering alts every day.`);
+			} else if (endquote === 2) {
+				this.add(`c|@Morfent ( _̀> ̀)|Every 60 seconds in Africa, a minute passes. Together we can stop this. Please spread the word.`);
+			} else {
+				this.add(`c|@Morfent ( _̀> ̀)|!dt morfent's husbando`);
+				this.add(`raw|<ul class="utilichart"><li class="result"><span class="col numcol">UU</span> <span class="col iconcol"><span class="picon" style="background: transparent url(&quot;//play.pokemonshowdown.com/sprites/smicons-sheet.png?a4&quot;) no-repeat scroll -400px -210px"></span></span> <span class="col pokemonnamecol" style="white-space: nowrap"><a href="https://pokemonshowdown.com/dex/pokemon/gengar" target="_blank" rel="noopener">Gengar</a></span> <span class="col typecol"><img src="https://play.pokemonshowdown.com/sprites/types/Ghost.png" alt="Ghost" height="14" width="32"><img src="https://play.pokemonshowdown.com/sprites/types/Poison.png" alt="Poison" height="14" width="32"></span> <span style="float: left ; min-height: 26px"><span class="col abilitycol">Cursed Body</span><span class="col abilitycol"></span></span><span style="float: left ; min-height: 26px"><span class="col statcol"><em>HP</em><br>60</span> <span class="col statcol"><em>Atk</em><br>65</span> <span class="col statcol"><em>Def</em><br>60</span> <span class="col statcol"><em>SpA</em><br>130</span> <span class="col statcol"><em>SpD</em><br>75</span> <span class="col statcol"><em>Spe</em><br>110</span> <span class="col bstcol"><em>BST<br>500</em></span> </span></li><li style="clear: both"></li></ul>`);
```

Comment:
dw bout that, the code works
![screenshot 77](https://user-images.githubusercontent.com/28816776/45933681-775a3900-bf5f-11e8-914f-cfc3a711719a.png)